mwdb/resources/oauth.py

```diff
@@ -520,6 +552,11 @@ def post(self, provider_name):
         if not provider:
             raise NotFound(f"Requested provider name '{provider_name}' not found")
 
+        group_name = provider_name + "_Open_ID_Provider"
```

Comment:
Maybe it should be common function e.g. on `model.OpenIDProvider` level?
```suggestion
        group_name = "OpenID_" + provider_name
```